packages/rancher-cis-benchmark/charts/templates/cis-roles.yaml

```diff
@@ -12,6 +12,17 @@ rules:
       - clusterscans
       - clusterscanreports
     verbs: ["create", "update", "delete", "patch","get", "watch", "list"]
+  - apiGroups:
+      - catalog.cattle.io
+    resources: ["apps"]
```

Comment:
Why is a v2 feature chart interacting with a v1 app ? If that’s what this is doing

Reply:
This is the cis app install config resource that Dashboard asked access for here: https://github.com/rancher/dashboard/issues/2020#issuecomment-741924379


Also seems like it is a V2 chart CRD:
```
> kubectl get apps.catalog.cattle.io -A
NAMESPACE                 NAME                        CHART                       VERSION                                                                  RELEASE NAME                RELEASE VERSION   STATUS
cattle-system             rancher                     rancher                     2.5.3                                                                    rancher                     2                 deployed
cattle-system             rancher-webhook             rancher-webhook             0.1.0-beta800                                                            rancher-webhook             2                 deployed
cert-manager              cert-manager                cert-manager                v0.15.0                                                                  cert-manager                1                 deployed
cis-operator-system       rancher-cis-benchmark       rancher-cis-benchmark       1.0.300                                                                  rancher-cis-benchmark       1                 deployed
cis-operator-system       rancher-cis-benchmark-crd   rancher-cis-benchmark-crd   1.0.300                                                                  rancher-cis-benchmark-crd   1                 deployed
fleet-system              fleet                       fleet                       0.3.100                                                                  fleet                       2                 deployed
fleet-system              fleet-agent                 fleet-agent                 v0.0.0+s-730feb12c5e7537f2ddfce6566108c5bb71f07813b653f4f9af666ce4720f   fleet-agent                 1                 deployed
fleet-system              fleet-crd                   fleet-crd                   0.3.100                                                                  fleet-crd                   1                 deployed
rancher-operator-system   rancher-operator            rancher-operator            0.1.100                                                                  rancher-operator            1                 deployed
rancher-operator-system   rancher-operator-crd        rancher-operator-crd        0.1.100                                                                  rancher-operator-crd        1                 deployed
```